tests/functional/pv/pvc_snapshot/test_pvc_snapshot.py

```
@@ -88,19 +87,22 @@ def test_pvc_snapshot(self, interface, teardown_factory, cluster_index):
         assert err_count == 0, (
             f"IO error on pod {self.pod_obj.name}. " f"FIO result: {fio_result}"
```

Ruff (0.15.15)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)